editor/inspector_dock.cpp

```diff
@@ -141,13 +141,14 @@ void InspectorDock::_menu_option_confirm(int p_option, bool p_confirmed) {
 
 				unique_resources_list_tree->clear();
 				if (resource_propnames.size()) {
-					TreeItem *root = unique_resources_list_tree->create_item();
+					const EditorPropertyNameProcessor::Style name_style = EditorPropertyNameProcessor::get_default_inspector_style();
```

Comment:
I mean, it should be literally `property_style_name`. The inspector dock uses `get_default_inspector_style()` as its default style on start up, but the style can be runtime overridden via the dropdown menu.